packages/web/src/js/Tooltip.ts

```diff
@@ -22,6 +22,7 @@ const SELECTOR_ARROW = '[data-spirit-element="arrow"]';
 const SELECTOR_WRAPPER = '[data-spirit-element="tooltip-wrapper"]';
 const CLASS_NAME_VISIBLE = 'is-visible';
 const CLASS_NAME_HIDDEN = 'is-hidden';
+const CLASS_NAME_DISMISSIBLE = 'Tooltip--dismissible';
```

Comment:
shouldn't we add is-dismissible class 🤔?

Reply:
reworked to check if there is data-spirit-dismiss="tooltip" in the children elements